src/ui/ResultAttachments/ResultAttachments.ts

```diff
@@ -30,39 +31,77 @@ export class ResultAttachments extends Component {
    * @componentOptions
    */
   static options: IResultAttachmentsOptions = {
+
     /**
-     * Specifies the template to use to render each of the attachments for a top result.<br/>
-     * By default, it will use the template specified in a child element with a `<script>` tag.<br/>
-     * This can be specified directly as an attribute to the element, for example :
+     * Specifies the template to use to render each attachment for a top result.
+     *
+     * You can specify a previously registered template for this option by referring the HTML `id` attribute or a CSS
+     * selector for the template (see {@link TemplateCache}).
+     *
+     * **Examples:**
+     *
+     * Specifying a previously registered template by referring to its HTML `id` attribute:
+     *
+     * ```html
+     * <div class="CoveoResultAttachments" data-result-template-id="Foo"></div>
+     * ```
+     *
+     * Specifying a previously registered template by referring to a CSS selector:
+     *
      * ```html
-     * <div class="CoveoResultFolding" data-result-template-id="Foo"></div>
+     * <div class='CoveoResultAttachments' data-result-template-selector=".Foo"></div>
      * ```
-     * which will use a previously registered template ID (see {@link TemplateCache})
+     *
+     * By default, this option uses the template specified in a child element with a `script` tag
      */
     resultTemplate: ComponentOptions.buildTemplateOption({ defaultFunction: (e) => new DefaultResultAttachmentTemplate() }),
+
     /**
-     * Specifies the template to use to render sub-attachments, which are attachments within other attachments,
-     * for example multiple files embedded in a .zip attachment.<br/>
-     * Sub-attachments can also contain other sub-attachments.<br/>
-     * The template can be specified the same way as {@link ResultAttachments.options.resultTemplate resultTemplate}
+     * Specifies the template to use to render sub-attachments, which are attachments within attachments, for example
+     * when multiple files are embedded within a .zip attachment.
+     *
+     * Sub-attachments can themselves sub-attachments, and so on up to a certain level (see
```

Comment:
`Sub-attachments can themselves sub-attachments` -> Looks like it missing a word